nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -899,8 +890,8 @@ in {
             mkExport = { arg, value }: "export ${arg}=${value}";
             dbpass = {
               arg = "DBPASS";
-              value = if c.dbpassFile != null
-                then ''"$(<"${toString c.dbpassFile}")"''
+              value = if cfg.secretFile != null
+                then ''"$(${lib.getExe pkgs.jq} -r '.dbpass // ""' "${toString cfg.secretFile}")"''
```

Comment:
While there are two alternatives to that (i.e. a password in settings - even though that's discouraged - and no password at all), I'm wondering if we should somehow point people to using that approach. That means
* writing docs
* perhaps even throwing a warning in a certain case, not sure how to implement that, though.
Ideas?

Reply:
So dbpass shoul be only added to the secrets option right? I could add a note to the documentation and maybe add a warning if dbpass is specified inside settings option?